src/test/java/com/epam/tc/hw1/TestDivide.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
```

Comment:
Please use different class
```suggestion
import static org.assertj.core.api.Assertions.assertThat;
```